megazords/cirrus/tests/python-tests/conftest.py

```diff
@@ -3,10 +3,20 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 import json
 import pytest
-from cirrus import CirrusClient
+from cirrus import CirrusClient, MetricsHandler, EnrollmentStatusExtraDef
 from fml import FmlClient
 
 
+class TestMetricsHandler(MetricsHandler):
+    recordings = []
+
+    @classmethod
+    def record_enrollment_statuses(self, enrollment_status_extras: [EnrollmentStatusExtraDef]):
```

Comment:
The correct annotation is `list[T]`
```suggestion
    def record_enrollment_statuses(self, enrollment_status_extras: list[EnrollmentStatusExtraDef]):
```